util/util.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"strconv"
+	"strings"
+)
+
+// Difference returns the difference between two arrays
+func Difference(arr1, arr2 []int64) (result []int64) {
+	hash := make(map[int64]bool)
+
+	for _, item := range arr1 {
+		hash[item] = true
+	}
+
+	for _, item := range arr2 {
+		if _, ok := hash[item]; !ok {
+			result = append(result, item)
+		}
+	}
+
+	return result
+}
+
+// SliceToString converts an int64 array to a comma separated string
+func SliceToString(arr []int64) string {
```

Comment:
I'm not a fan of *Slice*ToString. It doesn't really slice anything, does it?

Reply:
Slice is the name of the data structure[1]

[1] - https://tour.golang.org/moretypes/7